cmd/skupper/skupper_link.go

```diff
@@ -217,8 +221,9 @@ func NewCmdLinkStatus(skupperClient SkupperLinkClient) *cobra.Command {
 		},
 	}
 	cmd.Flags().IntVar(&waitFor, "wait", 0, "The number of seconds to wait for links to become connected")
-	cmd.Flags().DurationVar(&remoteInfoTimeout, "timeout", types.DefaultTimeoutDuration, "Configurable timeout for retrieving information about remote links")
 	cmd.Flags().BoolVar(&verboseLinkStatus, "verbose", false, "Show detailed information about a link")
+	cmd.Flags().DurationVar(&remoteInfoTimeout, "timeout", time.Second*5, "Configurable timeout for retrieving information about remote links")
+	cmd.Flags().BoolVar(&showRemoteLinks, "show-remote", false, "Show incoming links from other sites")
```

Comment:
Could we just use the verbose option to enable showing the incoming links? Else show-incoming-links would be clearer.

Reply:
Currently, the verbose option shows the details of a link (cost, etc). I can change it to show incoming links as a part of the verbose mode when listing all the links, but I also agree with you that `show-incoming-links` flag is clearer.